docs/intro/quickstart/workflow.rst

```diff
@@ -0,0 +1,111 @@
+.. _ref_quickstart_workflow:
+
+===============================
+A Smoother Development Workflow
```

Comment:
I think titles should be more descriptive that will result in better "searchability" (and SEO, ultimately), e.g. in this case: "Improving workflow with CLI hooks"